lib/getUrlsFromCrawler.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+module.exports = async (url, cacheTime) => {
+  return new Promise(function (resolve, reject) {
```

Comment:
If you have any async/await code inside the promise you would need to change it to 
```JavaScript
return new Promise(async function (resolve, reject) {
  ...
}
```

Reply:
Fixed